docs/how-to/enable_hugepages_for_upf.md

```diff
@@ -0,0 +1,38 @@
+# Enable HugePages in the UPF
+
+## Requirements
+
+- A Kubernetes Host 
+  - CPU that supports AVX2 and RDRAND instructions (Intel Haswell, AMD Excavator or equivalent)
+  - CPU has the flag "pdpe1gb" which allows to set 1Gi HugePages
+  - HugePages is enabled and has enough 2Gi free HugePages
+- A Kubernetes cluster with the Multus addon enabled
+- Juju = 3.1/stable
+
+## Deploy the UPF
+
+Deploy the UPF charm in a Juju model named `user-plane` by enabling HugePages.
+
+```bash
+juju add-model user-plane
+juju deploy sdcore-upf --trust --channel=edge --config enable-hugepages=True
+```
+
+## Validate HugePages support
+
+Check the bessd container logs which indicates that the HugePages are reserved.
+
+```bash
+kubectl logs -f  sdcore-upf-0 -c  bessd -n user-plane  | grep -i huge
+# Sample Output
+2023-10-26T12:33:35.495Z [bessd] I1026 12:33:35.495568    54 dpdk.cc:169] Initializing DPDK EAL with options: ["bessd", "--main-lcore", "127", "--lcore", "127@0-31", "--no-shconf", "--legacy-mem", "--socket-mem", "1024", "--huge-unlink"]
+2023-10-26T12:33:35.506Z [bessd] EAL: 48967 hugepages of size 2097152 reserved, but no mounted hugetlbfs found for that size
```

Comment:
I am not sure we want to show this log: in this case it seems it reserved huge pages of 2Mi instead of 1Gi ones. It happened to me one time when the host was not configured properly.

Reply:
I will modify the log as expected.